app/code/Magento/Sales/Test/Mftf/Test/AdminAvailabilityCreditMemoWithNoPaymentTest.xml

```diff
@@ -25,19 +25,19 @@
             </createData>
             <!-- Enable *Free Shipping* -->
             <createData entity="FreeShippingMethodsSettingConfig" stepKey="freeShippingMethodsSettingConfig"/>
-            <magentoCLI command="cache:flush" stepKey="flushCache"/>
+            <magentoCLI command="cache:clean config" stepKey="flushCache"/>
```

Comment:
Is it possible also rename stekKey to more meanfull?

Reply:
I'd love to do it. Sure, will commit with other improvements.